internal/server/handlers/k8s/database_engine.go

```diff
@@ -308,7 +308,7 @@ func preflightCheckDBEngineVersion(
 	if err != nil {
 		return false, "", err
 	}
-	return currentVersion.GreaterThanOrEqual(minVersion), minVersion.String(), nil
+	return currentVersion.GreaterThanOrEqual(minVersion), minVersion.Original(), nil
```

Comment:
`minVersion` itself is parsed correctly for comparison and internal use, but we need to return back the original form, not the internal representation used by the libaray